src/block-editor/filters/withBlockBinding.tsx

```diff
@@ -75,9 +75,10 @@ function BoundBlockEdit( props: BoundBlockEditProps ) {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody
-					title={ sprintf( __( 'Remote Data Block: %s', 'remote-data-blocks' ), remoteDataTitle ) }
-				>
+				<PanelBody title={ __( 'Remote Data', 'remote-data-blocks' ) }>
+					<p style={ { fontSize: 'small', color: 'rgb(117, 117, 117)' } }>
```

Comment:
The fontSize and colour are used to replicate the Gutenberg helper text shown on a lot of their components. Unfortunately, `PanelBody` doesn't have `help` as an option on it so this is the only way to show it.

I have set the fontSize to be small, and set the colour to be the same within the helper text. If anyone has a better suggestion, I'm happy to update this.

Reply:
Can this be targeted with a className instead of inline style?

Reply:
Done in the latest commit 👍🏾